src/lib/containers/entity/annotations/AnnotationEditorUtils.ts

```diff
@@ -42,56 +46,60 @@ export function getFriendlyPropertyName(error: AjvError) {
   }
 }
 
-export function transformErrors(errors: AjvError[]): AjvError[] {
-  // Transform the errors in the following ways:
-  // - Simplify the set of errors when failing to select an enumeration defined with an anyOf (SWC-5724)
-  // - Show a custom error message when using a property that collides with an internal entity property (SWC-5678)
+export function getTransformErrors(
+  concreteType: ENTITY_CONCRETE_TYPE = FOLDER_CONCRETE_TYPE_VALUE,
+) {
```

Comment:
Need the concrete type now, and the concrete type cannot be added as a regular argument because this function is passed to RJSF.

So, wrap the original function in a closure to capture the concrete type.